src/Documenter.jl

```diff
@@ -463,10 +439,7 @@ function deploydocs(;
             end
         end
     else
-        Utilities.log("skipping docs deployment.")
-        if get(ENV, "DOCUMENTER_DEBUG", "") != "true"
-              Utilities.log("You can set DOCUMENTER_DEBUG to \"true\" in Travis to see more information.")
-        end
+        @info "skipping docs deployment. Turn on debug output for more diagnostics (see the docs)."
```

Comment:
How do you actually do this easily in this case?

Reply:
`export JULIA_DEBUG=Documenter`